### DISCUSSION TITLE
Calling UseOpenTelemetryPrometheusScrapingEndpoint() hides vanilla prometheus metrics on endpoint

### DISCUSSION THREAD

**Discussion by shev7avl** [open-telemetry/opentelemetry-dotnet#5996] [Q&A]
I have a service, that uses default prometheus v5.0.x metrics for dotnet application. An example is down below

```
DotNetRuntimeStatsBuilder.Customize().WithGcStats().StartCollecting();

app.UseEndpoints(endpoints => {      
    endpoints.MapMetrics(); 
});
```

Now i need to enrich these metrics with native dotnet metrics (lets say from Npgsql library)

After registering needed metrics/view and calling 

```
app.UseOpenTelemetryPrometheusScrapingEndpoint()
```

the metrics from vanilla prometheus on /metrics endpoint are overridden with npgsql metrics

Is there any way or workaround to enrich prometheus metrics with OpenTelemetry metrics without mapping one of them to another endpoint?

Thanks in advance

**Comment by martincostello** [comment]
You can't see two competing implementations together.
